tests/test_instrument.py

```diff
@@ -205,7 +206,7 @@ def test_dimension_packer_config_defaults(self):
         """
         registry_config = RegistryConfig()
         registry_config["db"] = "sqlite://"
-        registry = Registry.createFromConfig(registry_config)
+        registry = SqlRegistry.createFromConfig(registry_config)
         self.instrument.register(registry)
```

Comment:
I suppose all our `Instrument` APIs are going to have to switch to using a `Butler` in the future.